release_build_files/readme.md

```diff
@@ -365,6 +365,8 @@ code.
 ### 8.0.0
     
 -   Changes
+    -   Analytics: Removed `SetCurrentScreen()` following the removal from underlying
+        native SDKs.
```

Comment:
Maybe add something about how it was deprecated from the native SDKs? Though it's a bummer that we didn't have it marked deprecated ourselves.